apps/emqx_web_hook/src/emqx_web_hook_actions.erl

```diff
@@ -20,43 +20,65 @@
-                              description => #{en => <<"The Content-Type of HTTP Request">>,
-                                               zh => <<"HTTP 请求头中的 Content-Type 字段值"/utf8>>}},
-            headers => #{order => 4,
-                         type => object,
-                         schema => #{},
-                         default => #{},
-                         title => #{en => <<"Request Header">>,
-                                    zh => <<"请求头"/utf8>>},
-                         description => #{en => <<"The custom HTTP request headers">>,
-                                          zh => <<"自定义的 HTTP 请求头列表"/utf8>>}}
+            url => #{
+                order => 1,
+                type => string,
+                format => url,
+                required => true,
+                title => #{en => <<"URL">>,
+                           zh => <<"URL"/utf8>>},
+                description => #{en => <<"The URL of the server that will receive the Webhook requests.">>,
+                                zh => <<"用于接收 Webhook 请求的服务器的 URL。"/utf8>>}
+            },
+            cacertfile => #{
+                order => 2,
+                type => file,
+                default => <<>>,
+                title => #{en => <<"CA Certificate File">>,
+                           zh => <<"CA 证书文件"/utf8>>},
+                description => #{en => <<"CA Certificate File.">>,
+                                 zh => <<"CA 证书文件。"/utf8>>}
+            },
+            certfile => #{
+                order => 3,
+                type => file,
+                default => <<>>,
+                title => #{en => <<"Certificate File">>,
+                           zh => <<"证书文件"/utf8>>},
+                description => #{en => <<"Certificate File.">>,
+                                 zh => <<"证书文件。"/utf8>>}
+            },
+            keyfile => #{
+                order => 4,
+                type => file,
+                default => <<>>,
+                title => #{en => <<"Private Key File">>,
+                           zh => <<"私钥文件"/utf8>>},
+                description => #{en => <<"Private key file.">>,
+                                 zh => <<"私钥文件。"/utf8>>}
+            },
+            verify => #{
+                order => 5,
+                type => boolean,
+                default => true,
+                title => #{en => <<"Verify">>,
+                           zh => <<"Verify"/utf8>>},
+                description => #{en => <<"Turn on peer certificate verification.">>,
+                                 zh => <<"是否开启对端证书验证。"/utf8>>}
+            },
+            pool_size => #{
+                order => 6,
+                type => number,
+                default => 8,
+                title => #{en => <<"Pool Size">>,
+                           zh => <<"连接池大小"/utf8>>},
+                description => #{en => <<"Pool Size for HTTP Server.">>,
+                                 zh => <<"HTTP Server 连接池大小。"/utf8>>}
```

Comment:
We should also merge the code for `connect_timeout` and `request_timeout` from 4.2.5 to here.